app/packages/spotlight/src/styles.module.css

```
@@ -28,7 +28,7 @@
   outline: none;
 }
 
-@-moz-document {
+@-moz-document url-prefix() {
```

Stylelint (17.14.0)

[error] Vendor-prefixed at-rule "`@-moz-document`" (at-rule-no-vendor-prefix)

(at-rule-no-vendor-prefix)